src/Hit/Cli.hs

```diff
@@ -89,7 +89,11 @@ freshP :: Parser HitCommand
 freshP = Fresh <$> maybeBranchP
 
 newP :: Parser HitCommand
-newP = New <$> issueNumP
+newP = New <$> issueNumP <*> assignOwnerP
+
+assignOwnerP :: Parser Bool
+assignOwnerP = switch
+    $ long "assign" <> short 'a' <> help "Assign yourself to issue"
```

Comment:
Also, how does it work if the issue number is not specified? Could we document this as well?

Reply:
Assigning a user to an issue is not a separate command of the entire program. It is instead a flag that can be passed into the `new` command. The `new` command requires the issue number and whether to assign the user to the issue or not (by default, no).